docs/guide/content_rendering/layout/add_login_form.md

```diff
@@ -0,0 +1,193 @@
+# Add user login form
+
+You can add a login form for your users. The template defines lots of config options for example, information about password expiration or [other user management templates](../../user_management/user_management.md#other-user-management-templates).
```

Comment:
```suggestion
You can add a login form for your users. The template defines lots of config options for example, information about password expiration or [other user management templates](../../user_management/user_management.md#other-user-management-templates).
```
The second sentence is unclear to me. Does the template *define* options, or can it use them?